com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -89,19 +93,28 @@ static AnimationParamEnumWrapper()
             }
         }
 
-        public override void OnDestroy()
+        protected void CleanUp()
```

Comment:
can we just "not" expose this API to keep this PR backportable to `release/1.0.0`?